ibm/service/power/data_source_ibm_pi_instances.go

```diff
@@ -267,3 +277,20 @@ func flattenPvmInstanceNetworks(list []*models.PVMInstanceNetwork) (networks []m
 	}
 	return
 }
+
+func flattenPvmInstanceFault(fault *models.PVMInstanceFault) map[string]interface{} {
+	faultMap := make(map[string]interface{})
+	if fault.Code != 0 {
+		faultMap[Attr_Code] = strconv.FormatFloat(fault.Code, 'f', -1, 64)
+	}
+	if !fault.Created.IsZero() {
```

Comment:
Please add a nil check.

Reply:
There are  nil check for fault like this one, which take care of that
`if powervmdata.Fault != nil`

Reply:
The fault.Created can be nil even if fault is not nil. In that case isZero will fail.

Reply:
created is of type `strfmt.DateTime` and if omitted, it should be default to zero dateTime.